go.mod

```
@@ -2,6 +2,8 @@ module github.com/minio/minio
 
 go 1.21
 
+replace github.com/minio/console => github.com/donatello/console v0.12.6-0.20240522161239-e2303ef9d681
```

```suggestion
replace github.com/minio/console => github.com/donatello/console v0.12.6-0.20240522161239-e2303ef9d681
```

You can point this to console master branch now.